### ISSUE TITLE
Dry run issue when passing parameter from one StepDef to another

### ISSUE THREAD

**Issue by bjuric** [gwen-interpreter/gwen#24]
The following causes Gwen to recurse forever trying to resolve a StepDef parameter value when executed in `--dry-run` mode only.

```
@StepDef
Scenario: I select "<a>" from the options list 
    Given I click option "$<a>" 
     When I do something with "$<a>"

@StepDef
Scenario: I do something with "<a>" 
```

In this example, `$<a>` resolves to `$<a>` at runtime when doing a dry run check on the call to the second StepDef (the `When` step). It hence infinitely recurses.

**Comment by bjuric** [comment]
Solution is to bind it to a decorated value for dry run purposes, by mapping `$<a>` to `$[param:a]` instead.

**Comment by bjuric** [comment]
Fixed in release [1.3.0](https://github.com/gwen-interpreter/gwen/releases/tag/v1.3.0)
